universe/UniverseGenerator.cpp

```diff
@@ -5,6 +5,7 @@
 #include "../util/i18n.h"
 #include "../util/Logger.h"
 #include "../parse/Parse.h"
+#include "../python/server/PythonServerFramework.h"
```

Comment:
I added that back when the max starlane length was passed as a call to a call to an interface function, which you correctly pointed out wasn't necessary, since it could just be an argument to GenerateStarlanes; I fixed it, but forgot to remove the line. So, please do remove the line.